frontend/mr-admin-flate/src/components/tilsagn/form/TilsagnForm.tsx

```diff
@@ -75,25 +76,36 @@ export function TilsagnForm(props: Props) {
               Tilsagn
             </Heading>
             <TwoColumnGrid separator>
-              <div className="pr-6">
-                <div className="grid grid-cols-2">
-                  <TextField
-                    size="small"
-                    label="Tilsagnstype"
-                    readOnly
-                    value={avtaletekster.tilsagn.type(tilsagnstype)}
-                  />
-                </div>
+              <VStack gap="6">
+                <TextField
+                  size="small"
+                  label="Tilsagnstype"
+                  readOnly
+                  value={avtaletekster.tilsagn.type(tilsagnstype)}
+                  className="max-w-fit"
+                />
                 {tilsagnstype === TilsagnType.INVESTERING && <InfomeldingOmInvesteringsTilsagn />}
-                <div className="py-3">
-                  <VelgPeriode startDato={gjennomforing.startDato} />
-                </div>
-                <div className="py-3">
-                  <VelgKostnadssted kostnadssteder={kostnadssteder} />
-                </div>
-                <Separator />
-                <div className="py-3">{props.beregningInput}</div>
-              </div>
+                <HGrid columns={2}>
+                  <ControlledDateInput
+                    label={tilsagnTekster.periode.start.label}
+                    fromDate={START_DATO_TILSAGN}
```

Comment:
Tror det er bedre om vi beholder startdato fra gjennomføring som "fromDate", selv om det for AFT- og VTA-tiltak kan bli langt flere valg enn strengt tatt nødvendig. For resten av tiltakene burde det være ok siden disse er ikke så langtlevende.